### ISSUE TITLE
failure to run source code after following instructions

### ISSUE THREAD

**Issue by Rad-dude** [suever/dicomsort#37]
Greetings,

I am on Mac OS X 10.13.6, I followed the steps described but when I try to run dicomsort after it fails.
Double checked that all requirements are satisfied but still.

This is the error message I get:
Ahmeds-MacBook-Pro:dicomSort radwan$ python DicomSort.pyw
Traceback (most recent call last):
  File "DicomSort.pyw", line 3, in <module>
    import gui
  File "/Users/radwan/Downloads/dicomSort/gui/__init__.py", line 10, in <module>
    import widgets
  File "/Users/radwan/Downloads/dicomSort/gui/widgets.py", line 87, in <module>
    class AboutDlg(wx.AboutDialogInfo):
AttributeError: 'module' object has no attribute 'AboutDialogInfo'

Any idea what's going wrong here ?

Thanks
Radwan

**Comment by suever** [comment]
Sorry that you're having some trouble with the installation. What version of wxpython are you using? Right now we only *officially* support version 3.x

```
import wx
print(wx.__version__)
```

I do have a branch locally that I have working with the alpha for WxPython Phoenix (4.x) that I can verify works with the latest stable version and push that as a branch for you to try out if you're using 4.x

**Comment by Heechberri** [comment]
Hi Suever,

I am currently using wxPython 4x

>import wx
>print(wx.__version__)
>4.0.4

and I also have problems running the source code following the instructions.

>$ python DicomSort.pyw 
>Traceback (most recent call last):
  >File "DicomSort.pyw", line 3, in <module>
    >import gui
  >File "/Users/revive_imac/dicomSort/gui/__init__.py", line 2, in <module>
    >import help
  >File "/Users/revive_imac/dicomSort/gui/help.py", line 1, in <module>
    >import wx.html
>ImportError: No module named wx.html

Any idea how to solve this?

Thank you!
Vae

**Comment by suever** [comment]
> ImportError: No module named wx.html

That error seems like the issue is that wxpython is not found at all, as that line is the first time that we import anything from `wx`.

That being said, I have a branch named `js_wxpython_4.0.4` in this repo that has preliminary support for wxpython 4.x. Feel free to try it out and let me know if you run into any issues.

**Comment by Heechberri** [comment]
Hi Suever,

Thanks for the reply. I have solved the problem. Yeap it certainly is a problem with python not being able to locate wxPython.

The problems is, I have python2.7 and python3.7.  "pip install" command is located in the /usr/local/lib/python3.7/site-packages/pip, and downloads wxPython into  /usr/local/lib/python3.7/site-packages/wxPython-4.0.4.dist-info. But when I run python it looks into usr/local/lib/python2.7/site-packages.  In the end I found  running pip as a module of python(2.7) worked for me and puts wxPython into the correct site-packages folder.

this is the command that I ran if it helps anyone 

> python -m pip install --user -U wxPython

I found out the problem when I tried to uninstall wxPython and it prompted me with 

> <img width="527" alt="Screenshot 2019-03-19 at 11 28 23 AM" src="https://user-images.githubusercontent.com/48549033/54585243-86283100-4a54-11e9-9f69-d9bb63922618.png">

which helped me to identify that the wxPython package was in the wrong folder.

Thank you for your help!!

Regards,
Vae
